xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -483,7 +488,7 @@
       </Parameters>
       <Docs>
         <param name="value">To be added.</param>
```

Comment:
@layomia @JeremyKuhne @ahsonkhan does the param value description suggested by @mairaw sound good? It would apply to all the method overloads.

Also, would you mind adding a return description? It's also missing.

Reply:
I added the extra params.

Reply:
Yep, looks good.